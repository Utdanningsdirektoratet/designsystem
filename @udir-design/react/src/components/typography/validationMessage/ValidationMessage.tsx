import { ValidationMessage } from '@digdir/designsystemet-react';

export { ValidationMessage };
export type { ValidationMessageProps } from '@digdir/designsystemet-react';
