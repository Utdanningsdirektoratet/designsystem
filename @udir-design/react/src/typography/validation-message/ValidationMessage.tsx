export {
  ValidationMessage,
  type ValidationMessageProps,
} from '@digdir/designsystemet-react';
