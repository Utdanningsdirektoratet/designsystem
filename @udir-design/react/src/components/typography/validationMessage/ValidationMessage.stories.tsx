import type { Meta, StoryObj } from '@storybook/react';
import { ValidationMessage } from './ValidationMessage';

const meta: Meta<typeof ValidationMessage> = {
  component: ValidationMessage,
};

export default meta;
type Story = StoryObj<typeof ValidationMessage>;

export const Preview: Story = {
  args: {
    children: 'ValidationMessage',
  },
};
