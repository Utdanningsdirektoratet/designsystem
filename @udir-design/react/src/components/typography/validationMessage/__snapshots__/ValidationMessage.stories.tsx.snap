// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Preview 1`] = `
<p data-field="validation">
  Dette er en valideringsmelding.
</p>
`;
